numba/array.py

```diff
@@ -0,0 +1,177 @@
+from numba.tracing import trace
+import numpy
+import sys
+
+def _o2s(dtype, shape, order):
+    # convert order parameter to strides
+
+    if dtype is None or shape is None or order is None:
```

Comment:
Right, but that's in the Array() call. For clarity, I don't think _o2s() should take such a signature.